source/sorted_table_model.ts

```diff
@@ -65,30 +66,82 @@ export class SortedTableModel extends TableModel {
   public constructor(source: TableModel, comparator?: Comparator,
       columnOrder?: ColumnOrder[]) {
     super();
+    this.translatedTable = new TranslatedTableModel(source);
+    if(comparator) {
+      this.comparator = comparator;
+    } else {
+      this.comparator = new Comparator();
+    }
+    if(columnOrder) {
+      this.order = columnOrder.slice();
+    } else {
+      this.order = [];
+    }
+    this.sort(source);
   }
 
   /** Returns the column sort order. */
   public get columnOrder(): ColumnOrder[] {
-    return null;
+    return this.order.slice();
   }
 
   /** Sets the order that the columns are sorted by. */
-  public set columnOrder(columnOrder: ColumnOrder[]) {}
+  public set columnOrder(columnOrder: ColumnOrder[]) {
+    this.order = columnOrder.slice();
```

Comment:
This is a bug, let me know whether you want to fix it in this case or a future case.

Reply:
I see the bug. I think it's better for it to be its own case.